packages/patterns-react/src/internal/FeatureFlags.js

```diff
@@ -39,3 +39,11 @@ export const LISTSECTION =
  */
 export const SIMPLELONGFORM =
   process.env.SIMPLELONGFORM === 'true' || DDS_FLAGS_ALL || false;
+
+/**
+ * This determines if the simplelongform will be rendered or not
+ *
+ * @type {string | boolean}
+ */
+export const DDS_CARDARRAY =
```

Comment:
We should be consistent if we're going to put underscores between words, right? I think `DDS_CARD_ARRAY` looks easier to read. Anyone else agree?

Reply:
I agree, this was discussed on #752. I'll use it as default from now on.